### ISSUE TITLE
docs-bug(Dialog): Highlighting bug on Chrome

### ISSUE THREAD

**Issue by mcmessina** [angular/components#19529]
#### Documentation Feedback

While following the example to create a Dialog box with an input area, clicking around the component causes strange highlighting effects in Chrome (see animation of bug below). I am using Chrome Version 83.0.4103.61 (Official Build) (64-bit) with MacOS Version 10.15.5 (19F101). I do not see this issue occurring on Safari or Firefox.

Is there a workaround to prevent this behavior that would require an update to the documentation, or is this perhaps a Chrome/Angular bug instead?

[![Bug animation][1]][1]

  [1]: https://i.stack.imgur.com/mRuHE.png

**Affected documentation page:**
[https://material.angular.io/components/dialog/overview](https://material.angular.io/components/dialog/overview)

**Comment by crisbeto** [comment]
I'm not seeing the issue on Chrome 83 on Windows. This might be some weird OS issue.

**Comment by mcmessina** [comment]
> I'm not seeing the issue on Chrome 83 on Windows. This might be some weird OS issue.

Thanks, I think you're right. The issue was persistent on my machine until today, but the issues is not occurring for me anymore. After I opened the issue I then saw similar blinking/highlighting effects on other html elements like checkboxes on other pages, so I agree with you and think this is a Chrome or OS issue--it does not seem to be an Angular Component bug. Apologies for the false alert. I'll close the issue. Thanks to all for the investigation and help.
